packages/xo-web/src/common/intl/messages.js

```diff
@@ -1331,6 +1332,8 @@ const messages = {
   vmCoresPerSocketExceedsSocketsLimit: 'The selected value exceeds the sockets limit ({maxSockets, number})',
   vmHaDisabled: 'Disabled',
   vmMemoryLimitsLabel: 'Memory limits (min/max)',
+  vmUuid: 'VM UUID',
+  vmUuidNotMandatory: 'VM UUID is not mandatory when specifying the address but it is still recommended to fill it',
```

Comment:
optionalVmUuid: VM UUID is optional but recommended.

or just add `optionalEntry` message